src/Cli/dotnet/commands/dotnet-workload/WorkloadCommandParser.cs

```diff
@@ -15,14 +20,68 @@ internal static class WorkloadCommandParser
 
         private static readonly Command Command = ConstructCommand();
 
+        public static readonly Option<bool> InfoOption = new Option<bool>("--info");
+
         public static Command GetCommand()
         {
+            Command.AddOption(InfoOption);
             return Command;
         }
 
+        internal static void ShowWorkloadsInfo(IWorkloadInfoHelper workloadInfoHelper = null, IReporter reporter = null)
+        {
+            workloadInfoHelper ??= new WorkloadInfoHelper();
+            IEnumerable<WorkloadId> installedList = workloadInfoHelper.InstalledSdkWorkloadIds;
+            InstalledWorkloadsCollection installedWorkloads = workloadInfoHelper.AddInstalledVsWorkloads(installedList);
+            reporter ??= Cli.Utils.Reporter.Output;
+
+            if(!installedList.Any())
+            {
+                reporter.WriteLine(CommonStrings.NoWorkloadsInstalledInfoWarning);
+                return;
+            }
+
+
+            foreach (var workload in installedWorkloads.AsEnumerable())
+            {
+                reporter.WriteLine("\n");
+                reporter.WriteLine(CommonStrings.WorkloadIdColumn + " : [" + workload.Key + "]");
```

Comment:
Extra space between the `:` and value on the left. 

![image](https://user-images.githubusercontent.com/7409981/178404857-b90ac319-1ec9-4102-be96-22b76c44bba9.png)